### ISSUE TITLE
comments of cards inside same stack are always added to the latest created and focused card.

### ISSUE THREAD

**Issue by andreElrico** [nextcloud/deck#958]
**Describe the bug**
see in title

**To Reproduce**
(~~1. deck -> create board: b1 -> create stack: s1
2. create card: s1_c1
3. create card: s1_c2
4. navigate to card s1_c2
5. navigate to card s1_c1 -> timeline -> add a "new comment": c1_c1
6. The comment c1_c1 is not shown on timeline of s1_c1, but on the timeline of s1_c2.~~)

[YOUTUBE_VIDEO](https://www.youtube.com/watch?v=e1vssN8OHMA)

**Expected behavior**
The comment c1_c1 should show on the timeline of card "s1_c1", because we posted it on card s1_c1.

nextcloud: Version: 15.0.5
deck: 0.5.2

PS: I can make a Video of the bug, if this is hard to follow.

**Comment by juliushaertl** [comment]
Thanks for your report. I tried to reproduce this with the steps from above but it works just fine here. Maybe the video can give some more insights. Can you try to record it with the browser console (in network tab) open like this: 
![image](https://user-images.githubusercontent.com/3404133/54985979-195fea00-4fb2-11e9-830c-e444c5e1b41a.png)

**Comment by bpcurse** [comment]
Could be related to #875

**Comment by andreElrico** [comment]
My description was not precise. Please watch the video: ( https://www.youtube.com/watch?v=e1vssN8OHMA )  (I have also added it to my first post.)
